cmd/server-main.go

```diff
@@ -343,18 +324,23 @@ func serverMain(c *cli.Context) {
 	serverAddr := c.String("address")
 
 	// Check if requested port is available.
-	host, port := splitHostPort(serverAddr)
-	fatalIf(checkPortAvailability(port), "Port unavailable %d", port)
+	host, portStr, err := net.SplitHostPort(serverAddr)
```

Comment:
actually previously it was getPort(), but i introduced splitHostPort and later removed it and used net.SplitHostPort directly.

Because by default we use port 9000, and if `--address "10.1.10.1" ` is used the default port gets changed to 80/443 this would be inconsistent behavior. it will be easy for the users to remember that if port is not specified it defaults 9000 always. right?